features/image-transformations/image-enhancement-and-color-manipulation.md

```diff
@@ -133,3 +133,117 @@ URL - [https://ik.imagekit.io/demo/lipstick_2M.png](https://ik.imagekit.io/demo/
 {% endtab %}
 
 {% endtabs %}
+
+### Gradient - (e-gradient)
+
+This is used to add a gradient overlay over an input image.
+
+The gradient formed is a *linear gradient* (containing two colors) and it can be customised with the following parameters:
+
+| Parameter       | Description       | Value Type        | Range      | Default Value     |
+|-----------------|-------------------|-------------------|------------|-------------------|
+| Linear Direction (`ld`) | **Optional**.<br/><br/>This is used to specify the direction vector of the linear gradient.<br/><br/>For enhanced control over the direction, this parameter also accepts a integer `0-359`, which specifies the angle (in degrees) when measured over a standard x-y axes with pivot passing through the geometrical centre of the image. For example, `0` would mean `top`, `45` would mean `top_right`, `90` would mean `right` etc. | `Number` or `String` | **Number** - `0` to `359`<br/><br/>**String** - `top`, `top_right`, `right`, `bottom_right`, `bottom`, `bottom_left`, `left`, `topleft` | `180` (or `bottom`) |
+| From Color (`from`) | **Optional**.<br/><br/>This is used to specify a *from* color for the gradient.<br/><br/>RGBA Code (e.g. `FFAABB50`) is a color representation model where the first six characters must be the RGB Hex Code and the last two characters must be a number between `00` and `99`, which indicates the opacity level. For example, `00` represents an opacity level of `0.00`, `01` represents an opacity level of `0.01`, and so on. | `String` | An [SVG recognized color](https://www.w3.org/TR/SVG11/types.html#ColorKeywords) keyword name eg. `aqua`;<br/><br/>or a RGB Hex Code eg. `FF0000`;<br/><br/>or a RGBA Hex Code eg. `FF000040`; | `FFFFFF` |
```

Comment:
`This is used to specify a *from* color for the gradient.`

This line isn't actually describing anything. Can it be rewritten to include more information about the parameter?

Reply:
Any suggestions? I can't think of anything more that would be needed to expand this.

Reply:
It can have the keywords `start` or `starting` maybe. Something like "the starting point of the gradient will have this color in its purest form"

Reply:
Done

Reply:
resolved